File: frontend/src/app/shared/models/issue/issue.ts
import { IssueStatus } from '@shared/models/issue/enums/issue-status';
import { Project } from '@shared/models/projects/project';

export interface Issue {
    id: number,
    errorMessage: string,
    errorClass: string,
    status: IssueStatus,
    project: Project,
}

File: backend/Watchdog.Core/Watchdog.Core.DAL/Entities/Issue.cs
using System.Collections.Generic;
using Watchdog.Core.Common.Enums.Issues;
using Watchdog.Core.DAL.Entities.Common;

namespace Watchdog.Core.DAL.Entities
{
    public class Issue : Entity<int>
    {
        public Issue()
        {
            EventMessages = new List<EventMessage>();
        }

        public string ErrorClass { get; set; }
        public string ErrorMessage { get; set; }
        public IssueStatus Status { get; set; }
        public Application Application { get; set; }
        public int ApplicationId { get; set; }
        public ICollection<EventMessage> EventMessages { get; set; }
    }
}